test/src/prayer_times_test.dart

```diff
@@ -329,14 +331,10 @@ void main() {
 
   test('Test PrayerTimes Invalid Coordinates', () {
     final invalidCoordinates = Coordinates(90.7750, -78.6336);
-    final prayerTimes = PrayerTimes.today(
-        invalidCoordinates, CalculationMethod.karachi.getParameters());
-    expect(prayerTimes.fajr, null);
-    expect(prayerTimes.sunrise, null);
-    expect(prayerTimes.dhuhr, null);
-    expect(prayerTimes.asr, null);
-    expect(prayerTimes.maghrib, null);
-    expect(prayerTimes.isha, null);
+    expect(
```

Comment:
We should ensure that prayerTimes.fajr, prayerTimes.sunrise and so on always have value, not nullable. We need to think of the nullable support from the ground. so everything makes sense.